index.html

```diff
@@ -116,7 +129,56 @@ <h2 class="recent-work">My Recent Works</h2>
             </div>
           </div>
           <div class="see-project">
-            <button class="invite one">See Project</button>
+            <button class="invite one" id="invite">See Project</button>
+          </div>
+          <div class="pop-up">
+            <div class="multi-mid">
+              <div>
+                <h3 class="pop-title">Multi Post Stories</h3>
```

Comment:
- [ ] Please make sure to add blury background around the popup window like given in Figma: 

Yours  | Figma
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/151182536-a8ae522f-4b52-4e94-9f48-14285f9b44aa.png)  | ![image](https://user-images.githubusercontent.com/77038610/151182444-763e2e13-3f45-48e8-b1a0-4aa60237623f.png)

As you can see the blury background in Figma serves for a better focus on popup content 😉 

- [ ] Please make the body{ overflow : hidden when the popup is visible, to prevent scrolling the page, because currently users can scroll the screen which should not be the case 😢